lib/src/utils.dart

```diff
@@ -240,9 +240,11 @@ class Utils {
         encoding: VideoEncoding(
           maxBitrate: math.max(
             150 * 1000,
-            (original.encoding.maxBitrate /
-                    (math.pow(scale, 2) *
-                        (original.encoding.maxFramerate / fps)))
+            (original.encoding?.maxBitrate ??
+                    VideoEncoding.defaults.maxBitrate /
```

Comment:
I think this is still a problem.. if it's not set.. then users would get stuck with a default encoding?

Reply:
We should be able to remove `VideoEncoding.defaults`. `encoding` in `original` will not be nil, because `_findAppropriateEncoding` will always assign it an appropriate one
![QQ_1723181742704](https://github.com/user-attachments/assets/8c673d13-f442-4cce-912f-7a7adc15470d)